lms/static/js/learner_dashboard/views/program_card_view.js

```diff
@@ -0,0 +1,32 @@
+;(function (define) {
+    'use strict';
+
+    define(['backbone',
+            'jquery',
+            'underscore',
+            'gettext',
+            'text!../../../templates/learner_dashboard/program_card.underscore'
+           ],
+         function(
+             Backbone,
+             $,
+             _,
+             gettext,
+             programCardTpl
+         ) {
+            return Backbone.View.extend({
+                className: 'program-card',
+                tpl: programCardTpl,
+                initialize: function(data) {
+                    this.render()
+                },
+
+                render: function() {
+                    var $el = $(this.el);
```

Comment:
Backbone Views already set ```this.$el``` which is the same as ```$(this.el)``` so you can use that instead of creating the jQuery object.